src/nix/get-env.sh

```diff
@@ -115,6 +115,16 @@ __dumpEnv() {
         printf '\n  }'
     fi
 
+    printf ',\n  "meta": {\n    '
+    # min-bash-version is supposed to be an integer.
+    # If not, just ignore it.
+    if [[ -n "${NIX_ENV_MIN_BASH_VERSION-}" && "$NIX_ENV_MIN_BASH_VERSION" -eq "$NIX_ENV_MIN_BASH_VERSION" ]]; then
```

Comment:
`NIX_ENV_MIN_BASH_VERSION` should be documented somewhere.

Reply:
Indeed. Any opinion on what the right place would be?

Reply:
I'd say like `nix-shell`, put it into [the command's docs](https://nixos.org/manual/nix/unstable/command-ref/nix-shell.html#environment-variables). That does mean the doc has to be duplicated between `print-dev-env` and `develop`, but it seems the most sensible place.